src/e2e/position.rs

```diff
@@ -72,9 +72,9 @@ fn test_create_position() {
 
 #[test]
 fn test_remove_position() {
-    let alice = test_env::get_account(0);
-    let bob = test_env::get_account(1);
-    test_env::set_caller(alice);
+    let deployer = test_env::get_account(0);
+    let user = test_env::get_account(1);
```

Comment:
avoid using just `user`. Name it to reflect the key operation that this account performs within the test. For example, in the test `test_remove_position` you can name this account `position_owner`